neqo-crypto/build.rs

```diff
@@ -12,8 +12,11 @@ use std::{
 };
 
 use bindgen::Builder;
+use semver::{Version, VersionReq};
 use serde_derive::Deserialize;
 
+include!("min_version.rs");
```

Comment:
I think that you can use `mod min_version; use min_version::MINIMUM_NSS_VERSION;` rather than relying on `include!()`.